tendrl/commons/message.py

```diff
@@ -0,0 +1,87 @@
+import datetime
+from inspect import getframeinfo
+from inspect import stack
+import json
+import uuid
+
+
+class Message(object):
+    # priority[0] is debug....priority[5] is critical
+    # Use priorities and publishers when call create message obj
+    priorities = type("priorities", (),
+                      {"DEBUG": "debug", "INFO": "info", "NOTICE": "notice",
+                       "WARNING": "warning", "ERROR": "error",
+                       "CRITICAL": "critical"})
```

Comment:
@r0h4n it is works like enum, for example when you wants to a message you need to give like this

Message(Message.priorities.DEBUG, ....) so it will pass value "debug" to __init__()